src/Backend/Modules/Users/Installer/Installer.php

```diff
@@ -181,7 +180,7 @@ private function loadGodUser(): void
             $group['user_id'] = $user['id'];
 
             // insert group
-            BackendGroupsModel::insertMultipleGroups($user['id'], [$group]);
+            $this->getDB()->insert('users_groups', $group);
```

Comment:
since you use 3 commits to first remove something and then add it back, can you rebase it and drop these 3 commits since they are clutter